extensions/amp-list/amp-list.md

```diff
@@ -249,6 +266,9 @@ Displays a loading indicator and placeholder again when the list's source is ref
 
 By default, this will only trigger on refreshes that cause a network fetch. To reset on all refreshes, use `reset-on-refresh="always"`.
 
+#### is-layout-container (experimental, optional)
```

Comment:
`[is-layout-container]`

Please also explain in the doc why this is only bindable and not available as a normal attribute (e.g. user action is needed for container since reflow, etc...)

Reply:
Done

Reply:
put it in `[]` since it is a bind-only attribute.